src/corehost/common/pal.unix.cpp

```diff
@@ -257,16 +257,27 @@ static
 pal::string_t normalize_linux_rid(pal::string_t rid)
 {
     pal::string_t rhelPrefix(_X("rhel."));
+    pal::string_t alpinePrefix(_X("alpine."));
+    size_t minorVersionSeparatorIndex = std::string::npos;
```

Comment:
calling this `minorVersionSeparatorIndex` is misleading now, since we use it for the patch version separator on alpine.

The same applies to the corresponding C# code below.

Reply:
I understand, I had the "minor" as the "last" version component in my head, but I can see that it is confusing.